docker/README.md

```diff
@@ -51,6 +52,40 @@ docker ps -a
 docker exec -it [container_name] /bin/bash
 ```
 
+## State Sync RPC Node
+
+Requirement: Follow the above steps to start a 4 node docker cluster before starting any state sync node
+
+```sh
+# Be sure to start up a 4-node cluster before you start a state sync node
+make docker-cluster-start
+# Wait for at least a few minutes till the latest block height exceed 500 (this can be changed via app.toml)
+seid status |jq
+# Start up a state sync node
+make run-rpc-node
+```
+
+## Local Debugging & Testing
+One of the fanciest thing of using docker is fast iteration. Here we support:
+- Being able to make changes locally and start up the chain to see the immediate impact
+- Being able to make changes to local dependency repo (Cosmo SDK/Tendermint) and start the chain with the latest changes without bumping or release any binary version
+
+
+In order to make local debugging work, you can follow these steps:
+```sh
+# Clone your dependency repo and put them under the same path as sei-chain
+cd sei-chain
+cd ../
+git clone https://github.com/sei-protocol/sei-tendermint.git
+git clone https://github.com/sei-protocol/sei-cosmos.git
+
+# Modify go.mod file to point to local repo, must use the exact same path as below:
+github.com/cosmos/cosmos-sdk => ../sei-cosmos
+github.com/tendermint/tendermint => ../sei-tendermint
```

Comment:
can use `go mod edit` to do this in an automated fashion

Reply:
oh interesting, didn't know that, let me try it out!